Makefile

```diff
@@ -13,6 +13,11 @@ IS_DOCKER_INSTALLED       := $(shell which docker >> /dev/null 2>&1; echo $$?)
 
 TARGETS := $(shell ls scripts)
 
+ifeq (${IMAGE_ORG}, )
+  IMAGE_ORG = openebs
```

Comment:
is this supposed to be ":="? seeing this at all the other places

Reply:
Checked this on. #TIL. 
When making direct string assignments, `:=` is not required. not required.  `:=`  is used in other places to expand expression during definition.

Reply:
both the forms does the expansion.. '=' makes recursive ones which can lead to infinite loop, where as, ':=' does expansion only once..
for direct assignments, both should be fine (https://www.gnu.org/software/make/manual/make.html#Flavors)